BUILD.md

```diff
@@ -53,17 +51,18 @@ The response should have some useful data.
 
```

Comment:
I don't know what is meant by this. We need to be either specific or just get rid of the RESP server testing part. I am not even sure how to exactly do this. If we want to keep it, let's give guidelines on how to do it.